src/plugins/intel_cpu/src/config.h

```diff
@@ -45,6 +45,11 @@ struct Config {
         PER_PLATFORM,
     };
 
+    enum class ModelType {
+        Convolution,
```

Comment:
minor: since this is model type enum better use "CNN" instead of just Convolution

Reply:
renamed